pkg/endpoint/endpoint.go

```diff
@@ -474,6 +476,14 @@ func (e *Endpoint) SetID() {
 	e.ID = e.IPv6.EndpointID()
 }
 
+func (e *Endpoint) GetIdentity() policy.NumericIdentity {
+	if e.SecLabel != nil {
+		return e.SecLabel.ID
+	}
+
+	return policy.NumericIdentity(0)
```

Comment:
Minor: the `policy.NumericIdentity(0)` could be its own const in pkg policy f.e. `InvalidNumericIdentity`, that is then reused also in e. writeHeaderfile() for the writing of SECLABEL{,_NB}, so that the relation is a bit more clear.